src/main/java/com/exactpro/epfast/template/implementations/ByteVectorFieldClass.java

```diff
@@ -0,0 +1,29 @@
+package com.exactpro.epfast.template.implementations;
+
+import com.exactpro.epfast.template.ByteVectorField;
+
+public class ByteVectorFieldClass extends FieldInstructionClass implements ByteVectorField {
+
+    private FieldOperatorClass fieldOperator;
+
+    private IdentityClass lengthFieldId = new IdentityClass();
+
+    @Override
+    public FieldOperatorClass getOperator() {
+        return fieldOperator;
+    }
+
+    @Override
+    public IdentityClass getLengthFieldId() {
+        return lengthFieldId;
+    }
+
+    public void setFieldOperator(FieldOperatorClass fieldOperator) {
```

Comment:
Rename to `setOperator`. Check that all get/set pairs are correct in other classes